src/indexer/handleEvent.ts

```diff
@@ -436,102 +419,126 @@ async function handleEvent(
 
     // --- Votes
     case "Voted": {
-      return async () => {
-        const voteId = ethers.utils.solidityKeccak256(
-          ["string"],
-          [`${event.blockNumber}-${event.logIndex}`]
-        );
+      const voteId = ethers.utils.solidityKeccak256(
```

Comment:
this event handler diff got all messed up, all  we're doing is removing the surrounding async closure, the rest is formatting.